libcontainer/SPEC.md

```diff
@@ -194,22 +202,40 @@ tree /sys/fs/resctrl
 |   |   |-- cbm_mask
 |   |   |-- min_cbm_bits
 |   |   |-- num_closids
+|   |-- L3_MON
+|   |   |-- max_threshold_occupancy
+|   |   |-- mon_features
+|   |   |-- num_rmids
 |   |-- MB
 |       |-- bandwidth_gran
 |       |-- delay_linear
 |       |-- min_bandwidth
 |       |-- num_closids
-|-- ...
+|-- mon_data
+    |-- mon_L3_00
+    	   |-- llc_occupancy
+    	   |-- mbm_local_bytes
+        |-- mbm_total_bytes
+    |-- ...
```

Comment:
How about removing these lines? For runc container use case, we don't touch the mon_data directory under root.